webhooks/collector.go

```diff
@@ -70,27 +73,91 @@ func collectHooks() []WebhookQueue {
 
 	for _, hook := range config.Config.Webhooks {
 
+		//TODO: check for area matching for each hook message
+
 		var totalCollection []WebhookMessage
 		if hook.Types == nil || slices.Contains(hook.Types, "gym") {
-			totalCollection = append(totalCollection, currentCollection[GymDetails].Messages...)
+			if hook.Areas == nil || len(hook.Areas) == 0 {
```

Comment:
len(nil array) in go is valid = 0
so you can just do `if len(hook.Areas) == 0` I would think